2/contrib/openshift/base-plugins.txt

```diff
@@ -1,23 +1,23 @@
 ant:1.11
 blueocean:1.24.8
-bouncycastle-api:2.23
+bouncycastle-api:2.25
```

Comment:
with the bump of dependencies in sync plugin I believe we can remove the explicit reference to this plugin, and let the plugin download script sort it out

I went through such exercises in the past when I was the primary maintainer

Let's remove and see what happens

Reply:
@gabemontero agreed; I wanted to question that as well.
And for some other dependencies, we can set unfixed versions as well.

I think that the probable reason it has been kept was probably for traceability and easily know which version of an important plugin we use instead of going look inside the RPM package.